css/css-grid/parsing/display-invalid.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Layout Module Level 1: parsing display with invalid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#propdef-display">
```

Comment:
This should link to `css-grid-1`, as we're testing Level 1 of the spec.

Note that the link is broken the right one would be something like:
https://drafts.csswg.org/css-grid-1/#ref-for-propdef-display%E2%91%A0
Which is quite strange, maybe this is an issue in the spec. :smile: